schemas/input/csl-data.json

```diff
@@ -468,8 +473,18 @@
         }
       ]
     },
+    "edtf-datatype": {
+      "title": "EDTF datatype pattern",
+      "description": "CSL input supports EDTF, validated against this regular expression.",
+      "type": "string"
```

Comment:
The regular expression would get added as a `pattern` property here.

Reply:
The regex to fully validate a EDTF might be pretty unwieldy. I think it's okay to not have a pattern or have one that checks only for major errors (like any invalid characters). But no strong opinion.